content/docs/operating/security.md

```diff
@@ -161,3 +161,16 @@ members of the Prometheus development team and the staff of those providers
 have access. If you are concerned about the exact provenance of your binaries,
 it is recommended to build them yourself rather than relying on the
 pre-built binaries provided by the project.
+
+## External audits
+
+[CNCF](https://cncf.io) sponsored an external security audit by
+[cure53](https://cure53.de) which ran from April 2018 to June 2018.
+
+The audit found concerns regarding Prometheus' secrity model, but following
```

Comment:
I don't think this really captures it, and this document covers everything mentioned.